src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -245,7 +245,7 @@ def _submit_bulk_run(
             run = self.run_operations.update(
                 name=run.name,
                 status=status,
-                end_time=datetime.datetime.now(),
+                end_time=datetime.datetime.now().astimezone(),
```

Comment:
why do we need to call astimezone() with no parameter?

Reply:
astimezone takes users timezone

Reply:
datetime.now() gives local date and time hence used `datetime.datetime.now().astimezone()` which gives same information but with timezone information included